helpers/deprecated/pick.js

```diff
@@ -1,13 +1,23 @@
 'use strict';
 
-const _ = require('lodash');
-
 const factory = () => {
     /**
      * @deprecate
      */
-    return function() {
-        return _.pick.apply(null, arguments);
+    return function(obj, ...args) {
+        return args.reduce(
+            (
+                pickedElements,
```

Comment:
can we just pass `obj` here ? and then at the bottom we do 
```
    obj[element] = obj[element];
```

Reply:
I think its probably better to avoid shadowing. Its not that we would be passing obj here, its what we would be naming the accumulator.